src/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <ProjectGuid>{13CE5E71-D373-4EA6-B3CB-166FF089A42A}</ProjectGuid>
     <SkipIncludeNewtonsoftJson>true</SkipIncludeNewtonsoftJson>
+    <LangVersion>7.2</LangVersion>
```

Comment:
Quesiton: Would we need to update the csproj  of other assemblies (like System.Memory) to include LangVersion 7.2 if we want to use the ref keyword i.e. here: https://github.com/dotnet/corefx/pull/23908/files?

Reply:
Yes, at least that was required for `ref structs` so for now I only did it in places where we used it. If we think we will want to use it everywhere, then we can set this property at the dir.props level later on so that it is not replicated on every project.

Reply:
This should be added to the top level props file instead.

Reply:
Yes lets try and move that to dir.props.

Reply:
@joperezr are you planning to move this to dir.props in a followup?

Reply:
If so, please also remove it from some of the places where I added it recently:
https://github.com/dotnet/corefx/pull/23908/files#diff-718e92a9d68dbfe993253e67ac1863f6R9